client/js/src/main/scala/zio/zmx/client/frontend/views/ChartView.scala

```diff
@@ -151,7 +151,7 @@ object ChartView {
         ),
         // This is the place holder for a form that will allow us to manipulate the settings of the Chartview
         div(
-          cls := "w-1/2 h-full p-3 ml-2",
+          cls := "w-1/3 h-full p-3 ml-2",
```

Comment:
Please keep that to `w-1/2`. The form should be a replacement for the string we initially put there